saleor/dashboard/order/forms.py

```diff
@@ -251,19 +251,22 @@ class OrderNoteForm(forms.Form):
         label=pgettext_lazy('Order note', 'Note'), widget=forms.Textarea())
 
 
-class ManagePaymentForm(forms.Form):
+class BasePaymentForm(forms.Form):
+
     amount = forms.DecimalField(
         label=pgettext_lazy(
             'Payment management form (capture, refund, void)', 'Amount'),
         max_digits=settings.DEFAULT_MAX_DIGITS,
         decimal_places=settings.DEFAULT_DECIMAL_PLACES)
 
+    clean_status = tuple()
+
     def __init__(self, *args, **kwargs):
         self.payment = kwargs.pop('payment')
         super().__init__(*args, **kwargs)
 
     def clean(self):
-        if self.payment.charge_status != self.clean_status:
+        if self.payment.charge_status not in self.clean_status:
             raise forms.ValidationError(self.clean_error)
```

Comment:
Since we're changing this form I think we could also define some base `clean_error` message. I know the subclasses implement it but it would be elegant if we provided a default one, what do you think?